utils.py

```diff
@@ -1092,8 +1070,9 @@ def decompress_from_zlib(data):
     return zlib.decompress(data)
 
 
-def compute_list_difference(list_a, list_b):
-    # type: (List[Any], List[Any]) -> List[Any]
+# The mentioned types can be changed in future if they are inadequate to
+# represent the types handled by this function.
+def compute_list_difference(list_a: List[str], list_b: List[str]) -> List[str]:
```

Comment:
Can we do 
```suggestion
def compute_list_difference(list_a: List[T], list_b: List[K]) -> List[Union[T, K]]:
```
where `K` would be another typevar (or it can be a different letter), I know we can be strict since it is only used in some places but I'm also interested if the solution above would work.

Reply:
@vojtechjelinek Yes it would work. Isn't the following a better option in that case?
```python
def compute_list_difference(list_a: List[T], list_b: List[T]) -> List[T]:
```

Reply:
Hmmm, okay. Just that I'm sure, my understanding of this error is that as we cannot be sure that `T` is comparable (if `T < T` is supported) thus we cannot do `sorted` on different of its sets. Can you try doing `CT = TypeVar('CT', bound=Comparable)` and using it here instead of `T`? Not sure if we should use this in the future, but I'm just interested in the result.

Reply:
I tried to use the solution suggested [here](https://stackoverflow.com/a/37669538) for defining Comparable and it works!
```python
from abc import ABCMeta, abstractmethod
from typing import Any, TypeVar

class Comparable(metaclass=ABCMeta):
    @abstractmethod
    def __lt__(self, other: Any) -> bool: ...

CT = TypeVar('CT', bound=Comparable)
```